dockerfiles/Dockerfile-ubuntu

```diff
@@ -16,14 +16,17 @@
 # specific language governing permissions and limitations
 # under the License.
 #
+
 # Copy this docker file to your local folder. Build the docker file like this - sudo docker build -t <username>/dispatch --file=Dockerfile-ubuntu . (don't miss the dot at the end)
 # To run it - sudo docker run -i -t <username>/dispatch (this will launch the dispatch router)
 
-# Gets the trusty Ubuntu from dockerhub
-FROM ubuntu:latest
+# Gets the Ubuntu from dockerhub
+FROM docker.io/library/ubuntu:focal
```

Comment:
Quick Q: Why do we need to include docker.io ? Can't we just say FROM ubuntu:focal ? I ran `podman search ubuntu --list-tags` and it looks like the only repo to house these ubuntu images is docker.io (for now). If any other repo houses these ubuntu images in the future, this can pull the image from there as well.

Otherwise this PR looks good.

Reply:
Done.

@ganeshmurthy I've somehow did not notice the comment sooner sorry.